code/game/gamemodes/dynamic/dynamic_simulations.dm

```diff
@@ -40,20 +40,55 @@
 	initialize_gamemode(config.forced_threat_level, config.roundstart_players)
 	create_candidates(config.roundstart_players)
 	gamemode.pre_setup()
+	gamemode.simulated = TRUE
```

Comment:
rather than doing this just make it a constructor argument to the gamemode

Reply:
Gamemodes are instantiated elsewhere so I don't want to do that since otherwise it would only make sense to add the argument to all gamemodes. Technically it would work because this is DM and DM doesn't care about typing things correctly but it would get a bit ugly